pkg/kubelet/kuberuntime/kuberuntime_image.go

```diff
@@ -111,42 +104,36 @@ func (m *kubeGenericRuntimeManager) PullImage(image kubecontainer.ImageSpec, pul
 // GetImageRef gets the ID of the image which has already been in
 // the local storage. It returns ("", nil) if the image isn't in the local storage.
 func (m *kubeGenericRuntimeManager) GetImageRef(image kubecontainer.ImageSpec) (string, error) {
+	if image.Pod == nil {
+		klog.Errorf("Pod is not set in imageSpec: %v", image)
+		return "", fmt.Errorf("POD is not set in imageSpec")
+	}
 
 	// Get the image service from the map cache for this image
```

Comment:
is this comment still applicable?

Reply:
no. it should be removed. will delete the comment.